gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/alttext/AltTextAction.kt

```diff
@@ -4,4 +4,6 @@ internal sealed class AltTextAction {
     data object AltTextUpdated : AltTextAction()
 
     data object AltTextUpdateFailed : AltTextAction()
+
+    data object AvatarCantBeLoaded : AltTextAction()
```

Comment:
This case shouldn't occur in the QE, but just in case and as a way to handle it.